src/Fl_Check_Browser.cxx

```diff
@@ -173,64 +172,64 @@ void Fl_Check_Browser::item_swap(void *a, void *b) {
 #define CHECK_SIZE (textsize()-2)
 
 int Fl_Check_Browser::item_width(void *v) const {
-        fl_font(textfont(), textsize());
-        return int(fl_width(((cb_item *)v)->text)) + CHECK_SIZE + 8;
+  fl_font(textfont(), textsize());
+  return int(fl_width(((cb_item *)v)->text)) + CHECK_SIZE + 8;
 }
 
 void Fl_Check_Browser::item_draw(void *v, int X, int Y, int, int) const {
-        cb_item *i = (cb_item *)v;
-        char *s = i->text;
-        int tsize = textsize();
-        Fl_Color col = active_r() ? textcolor() : fl_inactive(textcolor());
-        int cy = Y + (tsize + 1 - CHECK_SIZE) / 2;
-        X += 2;
-
-        fl_color(active_r() ? FL_FOREGROUND_COLOR : fl_inactive(FL_FOREGROUND_COLOR));
-        fl_loop(X, cy, X, cy + CHECK_SIZE,
-                X + CHECK_SIZE, cy + CHECK_SIZE, X + CHECK_SIZE, cy);
-        if (i->checked) {
-          int tx = X + 3;
-          int tw = CHECK_SIZE - 4;
-          int d1 = tw/3;
-          int d2 = tw-d1;
-          int ty = cy + (CHECK_SIZE+d2)/2-d1-2;
-          for (int n = 0; n < 3; n++, ty++) {
-            fl_line(tx, ty, tx+d1, ty+d1);
-            fl_line(tx+d1, ty+d1, tx+tw-1, ty+d1-d2+1);
-          }
-        }
-        fl_font(textfont(), tsize);
-        if (i->selected) {
-                col = fl_contrast(col, selection_color());
-        }
-        fl_color(col);
-        fl_draw(s, X + CHECK_SIZE + 8, Y + tsize - 1);
+  cb_item *i = (cb_item *)v;
+  char *s = i->text;
+  int tsize = textsize();
+  Fl_Color col = active_r() ? textcolor() : fl_inactive(textcolor());
+  int cy = Y + (tsize + 1 - CHECK_SIZE) / 2;
+  X += 2;
+
+  fl_color(active_r() ? FL_FOREGROUND_COLOR : fl_inactive(FL_FOREGROUND_COLOR));
+  fl_loop(X, cy, X, cy + CHECK_SIZE,
+          X + CHECK_SIZE, cy + CHECK_SIZE, X + CHECK_SIZE, cy);
+  if (i->checked) {
+    int tx = X + 3;
+    int tw = CHECK_SIZE - 4;
+    int d1 = tw / 3;
+    int d2 = tw - d1;
+    int ty = cy + (CHECK_SIZE + d2) / 2 - d1 - 2;
+    for (int n = 0; n < 3; n++, ty++) {
+      fl_line(tx, ty, tx + d1, ty + d1);
+      fl_line(tx + d1, ty + d1, tx + tw - 1, ty + d1 - d2 + 1);
```

Comment:
The use of condensed whitespace still seems clearer/more readable to me in the original code for the lines that use it, but I see it's used inconsistently.
If I'd been hand-editing the fixes, I'd probably have fixed only the indenting issues (as they were egregious, lol) and left the whitespace around the variables untouched, esp for the fl_line() functions. But perhaps consistency is best. Hard call.

Reply:
Personally I like the spaces around operators more because this is better readable for me (and ISTR that Ian said this also).

I left these here for consistency, as you wrote. I looked around in the file and found that this was the style of this file. I suggest to leave this as it is now.